libraries/AP_Mount/AP_Mount_Servo.cpp

```diff
@@ -60,10 +60,24 @@ void AP_Mount_Servo::update()
 
         // RC radio manual angle control, but with stabilization from the AHRS
         case MAV_MOUNT_MODE_RC_TARGETING:
-        {
-            // update targets using pilot's rc inputs
+        {   
+            // update targets using pilot's rc inputs or go to neutral or retracted targets if no rc
```

Comment:
This is only handling the servo backend, only one of many backends.

Reply:
right, none of the others use RC inputs for targets....it is conceivable that you might wish to have some kind of failsafe style behavior for the other backends, but its beyond the scope of this....your point at home would not probably, nor the neutral or retract backends, mavlink targetted probably not....gps targetted, perhaps